docs/Readme.md

```diff
@@ -0,0 +1,20 @@
+# React Native PowerAuth Mobile SDK
+
+In order to connect to the [PowerAuth](https://www.wultra.com/product/powerauth-mobile-security-suite) service, mobile applications need to perform the required network and cryptographic processes, as described in the PowerAuth documentation. To simplify the implementation of these processes, developers can use React Native library from this repository.
+
+## Integration Tutorials
```

Comment:
Use the "remove" macro here - this will be included in the sidebar and cause duplicity. A short, one-paragraph intro page for the docs is fine...

Reply:
Changed to `Equip your React Native application with Strong Customer Authentication (SCA) fast and easy.` I don't know, I'm not good with words :D

Reply:
The original text was OK. The comment was aimed towards the "table of contents" which is unnecessary and should be wrapped in `<!-- begin remove --><!-- end -->` macro...

Reply:
Oh, OK, changed...